test/integration/queue/producers/QueueProducer-test.js

```diff
@@ -14,7 +14,7 @@ const messageParser = require('~/queue/util/message-parser')
 const Event = require('~/models/events/Event')
 const waitForEvent = require('~/test/util/waitForEvent')
 
-const AMQ_URL = 'amqp://localhost'
+const AMQ_URL = 'amqp://rabbitmq'
```

Comment:
These should all be moved to some sort of test config. I'll handle this in a future PR since this is already getting too large.